src/components/AccessKeyForm.test.js

```diff
@@ -21,4 +26,65 @@ describe("Access Key Login Form", () => {
     );
     expect(container).toMatchSnapshot();
   });
+
+  it("shows list of sites if multiple redirects", () => {
```

Comment:
@zackkatz When testing locally, I did this same change to window.tlVendor to see what happens with multiple sites, that rendered a list of sites, when I click on one, it does a redirect as expected (not logged in since data is fake) and then I redid test with only one site in redirectData, that did a redirect right away.